config/locales/server.en.yml

```diff
@@ -23,7 +23,7 @@ en:
           Here are the results:
           %{table}
 
-          <a href='%{base_url}/admin/plugins/explorer?id=%{query_id}'>View query in Data Explorer</a>
+          <a href='%{base_url}/admin/plugins/explorer/queries/%{query_id}'>View query in Data Explorer</a>
```

Comment:
Maybe just check on Meta if we have Data Explorer docs there after you merge this, and if so update the links.

It's also likely that people have bookmarked these in a lot of places, so I think it's probably worth doing a 302 redirect on the server-side to go to the new endpoint

Reply:
Or if there was no server-side thing, perhaps do a clientside only redirect if they land on the /explorer route with an id?

Reply:
Yep, it was only client side, so I've added a redirect there.

Reply:
I've also had a read through the Data Explorer docs and tutorials on Meta, I couldn't see any references to the `?id=123` URL structure.